app/internal_packages/preferences/lib/tabs/workspace-section.jsx

```diff
@@ -145,7 +145,7 @@ const WorkspaceSection = props => {
       <div className="platform-note platform-linux-only">
         &quot;Launch on system start&quot; only works in XDG-compliant desktop environments. To
         enable the N1 icon in the system tray, you may need to install libappindicator1. (i.e.,
```

Comment:
shouldn’t that also read “Mailspring icon” instead of “N1 icon”?

Reply:
that's probably a good idea as well..

Do you want me to make the change in this pull request?